jcapi-systemusers.go

```diff
@@ -149,6 +160,16 @@ func getJCUserFieldsFromInterface(fields map[string]interface{}, user *JCUser) e
 			return err
 		}
 	}
+
+	if _, exists := fields["attributes"]; exists {
+		attributes := fields["attributes"].([]interface{})
+		for _, attributeMapInt := range attributes {
+			attributeMap := attributeMapInt.(map[string]interface{})
+			user.Attributes = append(user.Attributes,
+				JCUserAttribute{Name: attributeMap["name"].(string), Value: attributeMap["value"].(string)})
```

Comment:
Not sure how much of an issue it can ever really be, but we might want to verify that the `attributeMap` actually has the keys "name" and "value".